.github/workflows/tests.yml

```diff
@@ -4,7 +4,7 @@ on:
     branches: master
   pull_request: []
 jobs:
-  tests:
+  tests-macos:
```

Comment:
Would be good/more consistent with Homebrew/brew to use a single `tests` here with conditionals. I can do that if necessary!

Reply:
Initially I've done it like that, but then went for separate jobs.

Will do then.